include/rs_utils.h

```diff
@@ -12,6 +12,24 @@ char* rs_replace_all(
 
 void rs_cstring_free(char* str);
 
+class RustString {
+	private:
+		char* str;
+	public:
+		RustString() = delete;
+		RustString( const RustString & ) = delete;
+		RustString( const RustString && ) = delete;
```

Comment:
I deleted the move constructor because the issue mentions disallowing implicit construction. I can see why having it would be nice though. 
I'll add it in.